ngraph/test/provenance.cpp

```diff
@@ -601,89 +601,3 @@ TEST(provenance, opset0_downgrade_pass_topk)
     };
     traverse_nodes({topk_v0}, tag_check, as_node_vector(topk_v1->input_values()));
 }
-
-TEST(provenance, opset1_upgrade_pass_graph)
```

Comment:
The upgade pass is still used in the tests. It's supposed to be removed when we verify and confirm that the tests don't use `opset0` any more. I suggest to revert this change.

Reply:
I will recheck. Thank you!